diff --git a/gcc/Makefile.in b/gcc/Makefile.in
index 31ff95500c9..5b6023a399e 100644
--- a/gcc/Makefile.in
+++ b/gcc/Makefile.in
@@ -2775,7 +2775,6 @@ generated_files = config.h tm.h $(TM_P_H) $(TM_D_H) $(TM_H) multilib.h \
        $(ALL_GTFILES_H) gtype-desc.cc gtype-desc.h version.h \
        options.h target-hooks-def.h insn-opinit.h \
        common/common-target-hooks-def.h pass-instances.def \
-       gimple-match.cc generic-match.cc \
        c-family/c-target-hooks-def.h d/d-target-hooks-def.h \
        case-cfn-macros.h \
        cfn-operators.pd omp-device-properties.h
